src/bootstrap/configure.py

```diff
@@ -428,11 +429,18 @@ def configure_section(lines, config):
     else:
         configure_section(sections[section_key], section_config)
 
+@contextlib.contextmanager
+def output(filepath):
```

Comment:
`configure` imports `bootstrap`, so just reuse `bootstrap.output` and remove this def.

Reply:
Great point, fixed it accordingly.